plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -1799,14 +1800,14 @@ public void testS3SelectPushdown(String tableProperties)
                 .setCatalogSessionProperty("hive", "insert_existing_partitions_behavior", "APPEND")
```

Comment:
now that this class grew beyond its initial purpose,  this should be the class default and the insert-overwrite tests should pick OVERWRITE